vllm/config.py

```diff
@@ -374,13 +374,17 @@ def __init__(
         disable_custom_all_reduce: bool = False,
     ) -> None:
         self.pipeline_parallel_size = pipeline_parallel_size
-        self.tensor_parallel_size = tensor_parallel_size
+        if is_neuron():
```

Comment:
Please comment on why the `is_neuron` here is needed.

Reply:
comment added above.